<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCharacters</class>
 <widget class="QDialog" name="DialogCharacters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Characters</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelFont">
       <property name="text">
        <string>Font:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFontComboBox" name="comboBoxFont"/>
     </item>
     <item>
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSize"/>
     </item>
     <item>
      <widget class="QLabel" name="labelStyle">
       <property name="text">
        <string>Style:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxStyle"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxFontMerging">
       <property name="text">
        <string>Automatic font merging</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButtonProportional">
       <property name="text">
        <string>Proportional</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonMonospaced">
       <property name="text">
        <string>Monospaced</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAntialiasing">
       <property name="text">
        <string>Antialiasing</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>284</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogCharacters</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>716</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogCharacters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFont</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>DialogCharacters</receiver>
   <slot>findStyles(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxFont</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>DialogCharacters</receiver>
   <slot>findSizes(QFont)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>findStyles(QFont)</slot>
  <slot>findSizes(QFont)</slot>
 </slots>
</ui>
